pkg/controller/account/account_controller.go

```diff
@@ -607,14 +609,29 @@ func (r *ReconcileAccount) BuildIAMUser(reqLogger logr.Logger, awsClient awsclie
 	_, userErr := CreateIAMUser(reqLogger, awsClient, iamUserName)
 	// TODO: better error handling but for now scrap account
 	if userErr != nil {
-		failedToCreateIAMUserMsg := fmt.Sprintf("Failed to create IAM user %s", iamUserName)
-		SetAccountStatus(reqLogger, account, failedToCreateIAMUserMsg, awsv1alpha1.AccountFailed, "Failed")
-		err := r.Client.Status().Update(context.TODO(), account)
-		if err != nil {
-			return "", err
+		if aerr, ok := userErr.(awserr.Error); ok {
+			//If the user already exists, don't error
+			if aerr.Code() != "EntityAlreadyExists" {
```

Comment:
Other places we are using a case statement here where the below logic would go in `default:` could you update to match?

Reply:
:+1: